.claude/skills/beopsuny/scripts/fetch_law.py

```
@@ -1203,6 +1206,667 @@ def fetch_case_by_number(case_number: str):
+            print(f"🔍 {g['name']}")
+            print(f"   제목: {g['title']}")
+            print(f"   설명: {g['description']}")
+            print(f"   분류: {g['category']} | 질문 수: {g['item_count']}개")
+            print()
+
+    print("사용법: python scripts/fetch_law.py checklist show <name>")
+    print("예시: python scripts/fetch_law.py checklist show startup")
+    return checklists + guides
+
+
+def show_checklist(name: str, output_file: str = None, output_format: str = "markdown"):
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)